code/src/components/HoardingPlaceEntry.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './hoardingPlace.css'
+
+export const HoardingPlaceEntry = ({place, setPlace}) => {
+  return (
+    <form className="hoarding-place">
+      <h2>Where do you prefer to stock up?</h2>
+      <select className="hoarding-select"
+        onChange={event => setPlace(event.target.value)}
+        value={place} 
+      >
+        <option value=''>Hoarding Place:</option>
```

Comment:
Guess you don't want this as an option
```suggestion
        <option value='' disabled>Hoarding Place:</option>
```

Reply:
Ah, big thanks, very useful feedback!